internal/dataplane/parser/translators/httproute_test.go

```diff
@@ -33,6 +33,12 @@ func TestGeneratePluginsFromHTTPRouteFilters(t *testing.T) {
 								Value: "bar",
 							},
 						},
+						Add:  []gatewayv1beta1.HTTPHeader{
```

Comment:
As linter suggested, only one space should be reserved here.

Reply:
Thanks, I has just fix it.